ansible/install-apps.yml

```
@@ -0,0 +1,29 @@
+---
+- name: Install OS-level apps
+  hosts: all
+  become: yes
+  vars:
+  - apt_repo_list_file: /etc/apt/sources.list.d/google-chrome.list
+  tasks:
+  - stat:
+      path: "{{ apt_repo_list_file }}"
+    register: google_apt_exists
+    ignore_errors: yes
+  - name: dir
+    copy:
+       content: "{{ item }}"
+       dest: "{{ apt_repo_list_file }}"
+       owner: root
+       group: root
+       mode: 644
+    loop:
+     - "deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main"
```

plz keep the overall style consistent and use the same indentation level for sequences across the code base
```suggestion
    - "deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main"
```